dev-lang/odin/odin-2025.4.0.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="The Data-Oriented Language for Sane Software Development"
+DESCRIPTION="Odin is a general-purpose programming language with \
+distinct typing, built for high performance, modern systems, and \
+built-in data-oriented data types. The Odin Programming Language, \
+the C alternative for the joy of programming."
+HOMEPAGE="https://odin-lang.org/"
+COPYRIGHT="2016-2024 Ginger Bill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/odin-lang/Odin/archive/refs/tags/dev-2025-04.tar.gz"
+CHECKSUM_SHA256="c04f9adfa4d1f33110207ed2906a47b4f36a43232e5f09f9aed13e7ee4178894"
+SOURCE_DIR="Odin-dev-2025-04"
+
+ARCHITECTURES="all !x86_gcc2 ?x86"
+if [ "$targetArchitecture" = "x86_gcc2" ]; then
+	SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	odin$secondaryArchSuffix = $portVersion
+	cmd:odin$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libllvm$secondaryArchSuffix >= 20
+	lib:libiconv$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libllvm$secondaryArchSuffix >= 20
+	devel:libiconv$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:clang++$secondaryArchSuffix >= 20
+	cmd:llvm_config >= 20
+	cmd:awk
+	cmd:make
+	cmd:which
+	"
+
+TEST_REQUIRES="
+	"
+
+BUILD()
+{
+	make release-native
+}
+
+INSTALL()
+{
+	mkdir -p $prefix/bin
+
+	mkdir -p $prefix/lib/odin
+	cp -r odin base core shared vendor $prefix/lib/odin
```

Comment:
Did you check this on 32bit? this probably needs libDir instead of prefix/lib there? (not checked myself yet).

Reply:
no, always forget about checking for 32bits arch, will need to create a VM